source/gui/settingsDialogs.py

```diff
@@ -1882,6 +1882,17 @@ class ObjectPresentationPanel(SettingsPanel):
 		# See Progress bar output in the Object Presentation Settings section of the User Guide.
 		("both", _("Speak and beep")),
 	)
+	notificationLabels = (
+		# Translators: one of the accessible event notification choices
+		# to always announce notifications.
+		("always", _("always")),
+		# Translators: one of the accessible event notification choices
+		# to announce notifications from focused app.
+		("focusedApp", _("when application in foreground")),
+		# Translators: one of the accessible event notification choices
+		# to turn off notification announcement.
+		("off", pgettext("accessible event notification", "off"))

```

Comment:
I think it is better to put all these translations in the same category I.E. "accessible event notification"